### PR TITLE
Add layers package output for ECCO

### PR DESCRIPTION
**Pull request by cspencerjones**
This dataset contains the layers package output for ECCO and some variables like temperature in density coords.

### CODE REVIEW

File: intake-catalogs/ocean.yaml [hunk 1]
```diff
@@ -77,6 +77,21 @@ sources:
       storage_options:
         token: anon
       consolidated: True
+      
+   ECCO_layers:
```
Comment (rabernat) [inline intake-catalogs/ocean.yaml]: @cspencerjones  - I think the indentation here is wrong. That's why the CI is failing:

https://travis-ci.org/pangeo-data/pangeo-datastore/builds/572511324#L635

**Comment by cspencerjones** [comment]
This didn't solve it. I am worried that the problem is that the metadata isn't consolidated. Is that possible? Can I just change the flag to 'False'?

**Comment by rabernat** [comment]
That's correct, as you can see from the error log:
https://travis-ci.org/pangeo-data/pangeo-datastore/builds/611477893#L468

You can remove it, but it's a lot slower to load without. Could you instead add the consolidated metadata. It's easy to do: https://zarr.readthedocs.io/en/stable/tutorial.html#consolidating-metadata

**Comment by cspencerjones** [comment]
Am I supposed to be able to do this on pangeo, or do I need to do it locally and re-upload? When I try on pangeo, I get 'OSError: Forbidden: https://www.googleapis.com/upload/storage/v1/b/pangeo-data/o
Insufficient Permission'

**Comment by cspencerjones** [comment]
I'd really like it if this data wasn't deleted on Friday, so it'd be good if it can end up in the catalog. Maybe @naomi-henderson can help with my last question?

**Comment by naomi-henderson** [comment]
Hi @cspencerjones , here is how I would do it on one of my stores:
```
import gcsfs
import zarr
fs = gcsfs.GCSFileSystem(token='cache', access='read_write')
zstore = 'gs://cmip6/CMIP/IPSL/IPSL-CM6A-LR/1pctCO2/r1i1p1f1/Oyr/bsi/gn/'
zarr.consolidate_metadata(fs.get_mapper(zstore))  
```
and it put the correct .zmetadata file in the correct place

**Comment by cspencerjones** [comment]
Thanks Naomi! I'm having trouble with `fs = gcsfs.GCSFileSystem(token='browser', access='read_write')`. It gives me an error message when I go to the browser address:

`
401. That’s an error.

Error: deleted_client

The OAuth client was deleted.

Request Details
response_type=code
client_id=586241054156-ls4nduknhnelm2u6jtdgii15gsa3iv4v.apps.googleusercontent.com
redirect_uri=urn:ietf:wg:oauth:2.0:oob
scope=https://www.googleapis.com/auth/devstorage.read_write
state=khGQe6Ec2D39uYYI6XNyEyBHGyVTCX
prompt=consent
access_type=offline
That’s all we know.`

**Comment by naomi-henderson** [comment]
Hmmm,  I do this on my local machine, no problem, but if you on ocean.pangeo.io trying to run the code, you might need `token='cloud'`. See https://gcsfs.readthedocs.io/en/latest/api.html .

**Comment by cspencerjones** [comment]
OK, so I think it now is consolidated. I had terrible trouble getting authenticated and in the end used the .json credentials method. @rabernat can you run the tests again? Or do I need to do an empty push to get them to run?

**Pull request merged**